src/GitHub.VisualStudio.UI/Resources.resx

```diff
@@ -383,4 +383,10 @@
 
 [Don't show this again](dont-show-again)</value>
   </data>
-</root>
+  <data name="CompareFileWithWorkingDirectory" xml:space="preserve">
+    <value>Compare With Working Directory</value>
```

Comment:
Visual Studio seems to mainly use title casing even on articles and prepositions, for example "Edit -> Insert File As Text", "Edit -> Bookmark -> Previous Bookmark In Document" (although it is inconsistent - it uses e.g. "Attach to Process"). I agree though, it does look better to me the way you suggested...